src/features/swagger/components/Navigation/Navigation.tsx

```
@@ -0,0 +1,33 @@
+
+function Navigation(): React.ReactElement {
+  const router: NextRouter = useRouter();
+  const { t } = useTranslation();
+
+  const navigateHome: () => void = () => {
+    router.push('/');
+  };
+
+  return (
+    <Box sx={styles.navigationWrapper} onClick={navigateHome}>
+      <Stack direction="row" role="navigation" sx={styles.navigationButton}>
```

Biome

[error] The elements with the following roles can be changed to the following elements:
<nav>


For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)